.github/workflows/release-tag.yml

```diff
@@ -72,15 +72,22 @@ jobs:
     - name: Determine date
       id: date
       run: echo "::set-output name=value::$(date +%Y-%m-%d)"
-    - name: Generate CHANGELOG reference
-      uses: actions/github-script@v2
-      id: changelog_reference
+    - name: Get version from tag
+      id: version
+      uses: actions/github-script@v3
       with:
-        result-encoding: string
         script: |
-          const tag = context.ref.slice('refs/tags/v'.length);
-          const tagParts = tag.split('.');
-          require('fs').writeFileSync('/tmp/release-notes.md', `[Release notes](https://github.com/TheThingsNetwork/lorawan-stack/blob/${tag}/CHANGELOG.md#${tagParts[0]}${tagParts[1]}${tagParts[2]}---${{ steps.date.outputs.value }})`);
+          const tag = context.ref.slice("refs/tags/".length);
+          const majorMinorPatchRegex = /^v([0-9]+)\.([0-9]+)\.([0-9]+)$/;
```

Comment:
This doesn't actually work for `-rc` tags, but for those we already had incorrect results.

Perhaps we should just stop running this workflow on `-rc` tags?

Reply:
What means doesn't work, failure or wrong output?

I'm fine with running this workflow only on non-beta/rc/etc releases.

If we need a better way to distribute snapshots, we can do that, but not here.

Reply:
Ok, updated the tag filter so that it doesn't run.